### PR TITLE
use github actions instead of travis-ci.org

### PR DESCRIPTION
**Pull request by mpollmeier**

### CODE REVIEW

**Comment by mpollmeier** [comment]
ready for review, but only merge once secrets have been configured

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: PR
+on: pull_request
+jobs:
+  pr:
+    runs-on: ubuntu-18.04
```
Comment (hubertp) [inline .github/workflows/pr.yml:5]:
```suggestion
    runs-on: ubuntu-20.04
```
Let's stick to something a bit more recent?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Release
+on:
+  push:
+    branches: [master, main]
+    tags: ["*"]
+jobs:
+  release:
+    runs-on: ubuntu-18.04
```
Comment (hubertp) [inline .github/workflows/release.yml:8]:
```suggestion
    runs-on: ubuntu-20.04
```
same reason as before

**Review by hubertp — APPROVED** [review]
nits

**Review by fabsx00 — APPROVED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (mpollmeier) [reply .github/workflows/pr.yml:5]: From https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
image

It probably wouldn't harm, but..

Comment (mpollmeier) [reply .github/workflows/pr.yml:5]: discussed via slack, we'll stay with 18.04 for now

**Pull request merged**